3.7/debian-10/rootfs/opt/bitnami/scripts/libzookeeper.sh

```diff
@@ -70,8 +70,19 @@ zookeeper_validate() {
     check_allowed_port ZOO_PORT_NUMBER
     check_allowed_port ZOO_PROMETHEUS_METRICS_PORT_NUMBER
 
+    if is_boolean_yes "$ZOO_ADMIN_SERVER_ENABLE"; then
+      echo "Admin web server is enabled on port ${ZOO_ADMIN_SERVER_PORT_NUMBER}."
+      check_allowed_port ZOO_ADMIN_SERVER_PORT_NUMBER
+    else
+      echo "Admin web server is disabled."
+    fi
```

Comment:
Please check these suggestions:

```suggestion
is_boolean_yes "$ZOO_ADMIN_SERVER_ENABLE" && check_allowed_port ZOO_ADMIN_SERVER_PORT_NUMBER
```

In `_validate` functions we only make sure that the configuration is expected to work, not adding info messages about what will happen. That is meant for `_initialize`.

Reply:
Hey! Thank you for your review and suggestions!
This was refactored:
![image](https://user-images.githubusercontent.com/79898499/147945356-4c474967-0a17-4e24-a24a-579ab0e119c0.png)